src/shared/components/ncTable/mixins/rowHelper.js

```diff
@@ -0,0 +1,36 @@
+import { ColumnTypes } from './columnHandler.js'
+export default {
+	methods: {
+		isValueValidForColumn(value, column) {
+			const type = column?.type?.split('-')[0]
+			const columnTypeDefault = type + 'Default'
+			if (column.type === ColumnTypes.Selection) {
+				if (
+					(value instanceof Array && value.length > 0)
+					|| (value === parseInt(value))
+				) {
+					return true
+				}
+				const hasDefaultValue = columnTypeDefault in column && !(['', 'null'].includes(column[columnTypeDefault]))
+				return false || hasDefaultValue
```

Comment:
What's stopping from just returning `hasDefaultValue`? having the `false ||` seems redundant to me, since if `hasDefaultValue` is false, it'll just return false anyway. Unless I don't understand the logic correctly?